draft-ietf-quic-tls.md

```diff
@@ -1811,10 +1812,12 @@ EncryptedExtensions messages without the quic_transport_parameters extension
 MUST close the connection with an error of type 0x16d (equivalent to a fatal TLS
 missing_extension alert, see {{tls-errors}}).
 
-While the transport parameters are technically available prior to the completion
-of the handshake, they cannot be fully trusted until the handshake completes,
-and reliance on them should be minimized.  However, any tampering with the
-parameters will cause the handshake to fail.
+Transport parameters become available prior to the completion of the handshake.
+Using these values earlier than handshake completion is necessary at the server
```

Comment:
I'm not sure it's necessary to send 1-RTT packets prior to handshake completion, but it is necessary to send many things, such as data on streams in 0.5RTT packets.

Also, the way this is written, it could be read as sending 1-RTT packets from server to client requires using these values prior to handshake completion, which I realize isn't your intent.